### PR TITLE
Throw if Xata CLI not found

### PR DESCRIPTION
**Pull request by TejasQ**
## Summary

@SferaDev had great input in https://github.com/xataio/client-ts/pull/29#discussion_r826893541, so this PR is a subset of that one with this valuable change.

### CODE REVIEW

File: codegen/src/getCli.ts [hunk 1]
```diff
@@ -24,24 +24,19 @@ export const getCli = async ({ spinner }: { spinner: Ora }) => {
 `
   );
   spinner.start('Looking up latest Xata CLI...');
-  const fileUrl = await fetch('https://api.github.com/repos/xataio/cli/releases/latest')
-    .then((r) => r.json())
+  const fileUrl = await fetch('https://api.github.com/repos/xataio/cli/releases/skizzy')
+    .then(async (r) => {
```
Comment (gimenete) [inline codegen/src/getCli.ts:28]: Nitpick you can completely ignore. This function doesn't need to be async.

```suggestion
    .then((r) => {
```

File: codegen/src/getCli.ts [hunk 2]
```diff
@@ -24,24 +24,19 @@ export const getCli = async ({ spinner }: { spinner: Ora }) => {
 `
   );
   spinner.start('Looking up latest Xata CLI...');
-  const fileUrl = await fetch('https://api.github.com/repos/xataio/cli/releases/latest')
-    .then((r) => r.json())
+  const fileUrl = await fetch('https://api.github.com/repos/xataio/cli/releases/skizzy')
```
Comment (gimenete) [inline codegen/src/getCli.ts:27]: I guess you changed the URL to test the error scenario but forgot to put it back to the right URL.

```suggestion
  const fileUrl = await fetch('https://api.github.com/repos/xataio/cli/releases/latest')
```

**Review by gimenete — CHANGES_REQUESTED** [review]
One blocking comment. The URL is not correct. Other than that LGTM.

File: codegen/src/getCli.ts (hunk 1 above)
Comment (TejasQ) [reply codegen/src/getCli.ts:28]: Hahahahhaha really good catch. I was testing the failure case. 😂  Thank you for this!

**Review by gimenete — APPROVED** [review]
:shipit:

**Pull request merged**